private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -845,6 +913,132 @@ export class TemplateApi {
             });
         });
     }
+    /**
+     * 
+     * @summary Batch delete template version operation
+     * @param templateId ID of template to update
+     * @param versionList template versions to delete
+     * @param {*} [options] Override http request options.
+     */
+    public batchTemplateDelete (templateId: string, versionList: VersionList, options: any = {}) : Promise<{ response: http.IncomingMessage; body?: any;  }> {
+        const localVarPath = this.basePath + '/template/{templateId}/batch'
```

Comment:
batchTemplateDelete and batchTemplateUpdate are almost identical, maybe consider having a private method to be called by those?

Reply:
The files under typescript were autogenerated by the swagger file but I'll keep it in mind for final clean up!